playground/mocks/idp/idx/data/factor-verification-webauthn.json

```diff
@@ -0,0 +1,94 @@
+{
+    "stateHandle": "01pCiYqxWzm-iaFdL2W26KHFB-myLMJk2JBrLWmPxF",
+    "version": "1.0.0",
+    "expiresAt": "2019-08-14T20:42:04.000Z",
+    "step": "AUTHENTICATE",
+    "intent": "LOGIN",
+    "remediation": {
+      "type": "array",
+      "value": [
+        {
+          "rel": [
+            "create-form"
+          ],
+          "name": "challenge-factor",
+          "href": "http://localhost:3000/idp/idx/challenge",
+          "method": "POST",
+          "accepts": "application/vnd.okta.v1+json",
+          "value": [
+            {
+                "name": "clientData",
+                "required": true,
+                "visible": false
+            },
+            {
+                "name": "authenticatorData",
+                "required": true,
+                "visible": false
+            },
+            {
+                "name": "signatureData",
+                "required": true,
+                "visible": false
+            },
+            {
+              "name": "stateHandle",
+              "value": "01pCiYqxWzm-iaFdL2W26KHFB-myLMJk2JBrLWmPxF",
+              "visible": false
+            }
+          ]
+        }
+      ]
+    },
+    "factor": {
```

Comment:
Unlike other factors for webauhtn we need all enrolled factors for this profile in one object so we can pass to credentials.get method.

Reply:
may be have 'challenge' object which has enrolled factor and nonce info.